docs/faq.md

```diff
@@ -8,6 +8,12 @@ To upgrade Ollama, run the installation process again. On the Mac, click the Oll
 
 Review the [Troubleshooting](./troubleshooting.md) docs for more about using logs.
 
+## What are the components of Ollama that need to be running to work with the CLI, the API, and 3rd party tools?
```

Comment:
We can simplify this to "How can I manually run `ollama serve`?" Future FAQs below can follow suit (I added an example). This is definitely a missing FAQ

Reply:
I don't think we can simplify that in that way, because in order to know that you need to use ollama serve, you need to know that its one of the two components of running ollama

or did I misunderstand the comment